src/main/java/org/geysermc/discordbot/commands/DownloadCommand.java

```diff
@@ -83,12 +93,20 @@ protected void execute(CommandEvent event) {
 
         DownloadOption downloadOption = optionsToRepository.getOrDefault(program.toLowerCase(Locale.ROOT), this.defaultDownloadOption);
 
-        event.getMessage().replyEmbeds(new EmbedBuilder()
-                .setTitle("Download " + downloadOption.friendlyName)
-                .setDescription("Download at " + downloadOption.downloadUrl)
-                .setThumbnail(downloadOption.imageUrl)
-                .setColor(BotColors.SUCCESS.getColor())
-                .build()).queue();
+        event.getMessage().replyComponents(
+                        Container.of(
```

Comment:
You should make this a new method that returns this container, so you can just `.replyComponents(buildComponent())` and not duplicate the code.

We should have done this before, but since it's being refactored, we may as well do it now.

It can probably be something like
```java
private Container downloadContainer(DownloadOption downloadOption) {
   return Container.of(...);
}
```

Then just do `.replyComponents(downloadContainer(downloadOption))...`